src/jsonsync.py

```
@@ -0,0 +1,101 @@
+import json
+import os
+from threading import RLock
```

Ruff (0.12.2)

`threading.RLock` imported but unused

Remove unused import: `threading.RLock`

(F401)